### DISCUSSION TITLE
Alacritty (Often) Doesn't Redraw (Solved)

### DISCUSSION THREAD

**Discussion by Jamerman** [i3/i3#6327] [Q&A] [answered]
I've finally gotten around to setting up i3 on my Mint install. Polybar and Picom are all installed and configured, so everything's looking nice and ready to go.

### Alacritty Redrawing
The one issue I'm having is that Alacritty will occasionally not redraw. I did not have this issue at all on XFCE and seems to have been there from the start with i3. Say we do the old `sudo apt get update` and a load of text appears on the terminal. Once the command has run, I'll often have to type something into the terminal to get the screen to actually update. Maybe I'll be typing something in NeoVim and the odd input won't actually update the screen, but the next one will.

### What I'm going to try next
I'm going to try some different terminals and see if they behave the same way. If other terminals behave themselves, I'll remove/reinstall Alacritty and see if that helps. Maybe it just doesn't play well with tiling window managers, idk. I don't recall having this issue with the default xfce4 terminal when I first set i3 up and changed the terminal in the config file, but then I might not have used it long enough to notice. Also, though I *think* I've had this issue since the start of using i3, I can't remember if this happened before installing picom. Might be worth uninstalling that to test.

Has anyone else run into this issue? I've seen some reddit/github posts from a few years regarding this, but they all reported updates fixing everything.

**Comment by okraits** [comment]
Did you check if it happens as well if picom is not running?
I don't have this problem with alacritty and I didn't have it with other terminals either. I don't use a compositor, though.

**Reply by Jamerman** [reply]
You might be onto something there. I'll check that soon.

**Reply by Jamerman** [reply, marked as answer]
Found the problem! It was picom, after all. I didn't realise that it was already running on startup and had written this line into my i3config:

`exec --no-startup-id picom --config ~/.config/picom/picom.conf -b`

Removing that line fixed the issue. I guess that two instances of picom must have been running or something.
